README.md

```diff
@@ -1,2 +1,53 @@
-# _template
-Template repo with README and CI settings
+# LV SJPG DECODER 
+- (first commit)
+
+# Overview:
+  - SJPG is 'split-jpeg' which is a bundle of small jpeg fragments with an sjpg header.
+  - SJPG size will be almost comparable to the jpg file or might be a slightly larger.
+  - lv_sjpg can open and handle multiple files at same time.
+  - File read from disk (fread) and c-array is implemented.
+  - SJPEG frame fragment cache enables fast fetching of lines if availble in cache.
+  - By default the image cache will be xres * 2 * 16 bytes (can be modified)
+  - Currently only 16 bit image format is supported (to do)
+# Sample code:
+```c
+lv_obj_t * img1;
+lv_obj_t * img2;
+
+void lv_demo_sjpeg(void)
+{
+    lv_jpeg_init();
```

Comment:
It should be called `lv_split_jpg_init()` to differentiate it from a "normal" jpg decoder. I'll rename the repo to `lv_lib_split_jpg` too.

Reply:
Yes, lv_split_jpg_init makes more sense.

Reply:
Okay, I renamed the repo.